Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClientContextGlobal.cs

```diff
@@ -24,20 +24,15 @@ namespace Microsoft.PythonTools.LanguageServerClient {
     internal class PythonLanguageClientContextGlobal : IPythonLanguageClientContext, IDisposable {
         private readonly IInterpreterOptionsService _optionsService;
         private readonly DisposableBag _disposables;
-
         private IPythonInterpreterFactory _factory;
 
         public event EventHandler InterpreterChanged;
-
 #pragma warning disable CS0067
         public event EventHandler SearchPathsChanged;
         public event EventHandler Closed;
 #pragma warning restore CS0067
 
-        public PythonLanguageClientContextGlobal(
-            IInterpreterOptionsService optionsService,
-            string contentTypeName
-        ) {
+        public PythonLanguageClientContextGlobal(IInterpreterOptionsService optionsService, string contentTypeName) {
```

Comment:
Btw, the contentTypeName here and in other places is a leftover from MPLS where each language server instance had its own derived content type in order to associate a file with the correct server.

We no longer need this, so if you feel like it, you can remove it from here, and the IPythonLanguageClientContext interface.

Reply:
Yeah, I removed it. I was also wondering why it was there... :-)